File: produtos/src/main/java/br/com/bota/repository/ProdutoRepository.java
package br.com.bota.repository;

import br.com.bota.entity.Produto;
import io.micronaut.data.annotation.Repository;
import io.micronaut.data.jpa.repository.JpaRepository;

import java.util.UUID;

@Repository
public interface ProdutoRepository extends JpaRepository<Produto, UUID> {
}

File: loja-kotlin/kprodutos/src/main/kotlin/br/com/bota/repository/ProdutoRepository.kt
package br.com.bota.repository

import br.com.bota.entity.Produto
import io.micronaut.data.annotation.Repository
import io.micronaut.data.jpa.repository.JpaRepository
import java.util.*

@Repository
interface ProdutoRepository : JpaRepository<Produto, UUID>


